mock/api.js

```diff
@@ -281,21 +287,59 @@ export const mockSession = {
   },
 };
 
+export const mockEndpoints = {
```

Comment:
Hey Dave, what are your thoughts on whether we should keep `/api/v1` prefix for all the endpoints here since when this corresponds to mock? So, we can do something like /api/mock/controllers/list and change the corresponding handlers too

Reply:
So far as I can tell it doesn't make much difference as the mock mechanism seems to trigger based on a substring match; however, since you had to add the prefix to get the `fetch` from `index.html` to trigger, I thought it looked odd to have the others without the prefix so I added them everywhere. And that means the `get` or `post` from the dashboard code needs to include the prefix or it won't match ... so this makes the `mockEndpoints` JSON consistent with the `api.js` mocks.

The problem with changing to use a mocked path like `/api/mock` would be in that `document.ejs` (`index.html`) `fetch`; I think we'd need to go back to using a config variable to make EJS change the path based on whether we're building for test or production, or we'll break the `fetch('/api/v1/endpoints')` call again. I really liked the idea of making the URI a literal so there's no difference in the generated page.